src/java_bytecode/java_class_loader.h

```diff
@@ -51,22 +69,22 @@ class java_class_loadert:public messaget
       std::string class_file_name;
     };
 
-    // class name to index map
+    /// Maps class names to a jar file entry
```

Comment:
I also always wonder about it. The reason why I put them in the type is that the objects discussed by the comment are obvious there as the type arguments of the template. And because the type is anyway very close to the (single) field of this type.

Can you see reasons to document the field instead? Perhaps both?